tests/FeaturesTest.php

```diff
@@ -37,6 +38,12 @@ public function testFeatures(string $file): void
 
     private function analyze(string $file): int
     {
+        $laravelMajorVersion = (int) explode('.', app()->version())[0];
```

Comment:
I would rather have these checks in `getFeatures` method.

Reply:
It's not possible because the framework is not initialize yet in a data provider (I think it's the reason)

Reply:
Ok, good point! So let's leave it for now :+1: